canonical-web-and-design/tools.demos.haus
# Add new sizes of social banner

## Done

- Add new sizes of social banner requested [here](https://github.com/canonical-web-and-design/web-squad/issues/3827)
- Remove required from illustration field for smaller banners
- Update node-sass to sass

## QA

- View page at: https://tools-demos-haus-42.demos.haus
- Build a new banner (be sure to include an illustration) and check the resolutions including and following the Ad banner (728px × 90px) 

## Issue / Card

Fixes [#3827](https://github.com/canonical-web-and-design/web-squad/issues/3827)

## Screenshots

![image](https://user-images.githubusercontent.com/58959073/108729805-d2bef580-7522-11eb-978c-8c1ce03252f3.png)

![image](https://user-images.githubusercontent.com/58959073/108729816-d5b9e600-7522-11eb-8b9c-0b91597abbf8.png)

![image](https://user-images.githubusercontent.com/58959073/108729828-d8b4d680-7522-11eb-87a3-092145350f0e.png)

![image](https://user-images.githubusercontent.com/58959073/108729846-dbafc700-7522-11eb-9e61-ce302d3eefc9.png)
